src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -242,6 +242,10 @@
         - name: --enable-private-cluster
           type: string
           short-summary: Enable private cluster.
+        - name: --private-dns-zone
+          type: string
+          short-summary: (PREVIEW) private dns zone mode for private cluster.
```

Comment:
Please use `is_preview` attribute to make the parameter as Preview.

Reply:
seems I couldn't find such example for the parameter description here.

Reply:
`is_preview` would should help in a different format compared to other features. prefer `(PREVIEW)` to keep consistency.